src/PreCICE.jl

```diff
@@ -1,11 +1,17 @@
 module PreCICE
 """
 The `PreCICE` module provides the bindings for using the preCICE api. For more information, visit https://precice.org/.
-"""
+""" 
 
 
-# TODO add 'return nothing' keyword to void functions
+# These were proposed by the previous maintainer:
+# TODO add 'return nothing' keyword to void functions    # does this make sense? not done by documenter.jl
```

Comment:
Lets remove this if not found relevant

Reply:
I just found the [convention](https://docs.julialang.org/en/v1/manual/functions/#Returning-nothing) to add `return nothing` at the end of each function, so this should probably be done.

Reply:
Lets add this wherever relevant :+1: